providers/gcp.go

```diff
@@ -0,0 +1,62 @@
+package providers
+
+import (
+	"fmt"
+	captionsConfig "github.com/NYTimes/video-captions-api/config"
+	"github.com/NYTimes/video-captions-api/database"
+	log "github.com/Sirupsen/logrus"
+)
+
+// GCPProvider in a GCP client wrapper that implements the Provider interface
+type GCPProvider struct {
```

Comment:
Since there's nothing using GCP in this provider, should we rename it to ``UploadProvider``?